### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
Fixed a bug in pagination that was causing errors on pages past 5… then redid pagination to use Laravel’s built-in pagination and a custom presenter for Forge markup. I also added an API response collection class for simplifying dealing with “collections” returned from Northstar.

—
For review: @angaither

### CODE REVIEW

File: app/Http/Presenters/ForgePaginationPresenter.php [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+    /**
+     * Get HTML wrapper for active text.
+     *
+     * @param  string  $text
+     * @return string
+     */
+    protected function getActivePageWrapper($text)
+    {
+        return '<li class="is-active"><a href="#">'.$text.'</a></li>';
```
Comment (angaither) [inline app/Http/Presenters/ForgePaginationPresenter.php]: this doesn't look like :otwell:

**Comment by angaither** [comment]
:+1:

File: app/Http/Presenters/ForgePaginationPresenter.php (hunk 1 above)
Comment (DFurnes) [inline app/Http/Presenters/ForgePaginationPresenter.php]: Surprising [but true](https://github.com/laravel/framework/blob/e804393253a36d32161c95a7b37639cdb6009b75/src/Illuminate/Pagination/BootstrapThreePresenter.php#L103)!

Comment (angaither) [inline app/Http/Presenters/ForgePaginationPresenter.php]: well, actually. 
image

Comment (DFurnes) [inline app/Http/Presenters/ForgePaginationPresenter.php]: GOD DAMN IT, GRAHAM.

Comment (weerd) [inline app/Http/Presenters/ForgePaginationPresenter.php]: Yaaasss!

**Pull request merged**